mappings/net/minecraft/server/DummyFilter.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS none/me net/minecraft/server/DummyFilter
```

Comment:
I can change it, should I change the interface to IConnectionFilter as well then?